tests/conftest.py

```diff
@@ -82,6 +88,12 @@ def gprofiler_exe(request, tmp_path: Path) -> Path:
     return tmp_path / "gprofiler"
 
 
+def _print_process_output(popen: subprocess.Popen) -> None:
```

Comment:
if the process is hanging, this will hang as well.
add a timeout

Reply:
It's called only after kill/after it has exited